playbooks/network.yaml

```diff
@@ -99,41 +61,23 @@
     vars:
       checkpoint: "{{ (nmstateset.stdout_lines|last).split()[1] }}"
 
-  - name: Configure routing for dummy0 when used as public interface
-    block:
-    # NOTE(mdbooth): This task will spin indefinitely until the br-ctlplane
-    # interface is added. There must be a better way to do this. Adding this route
-    # to the dummy0 interface doesn't work: you can't route to floating IPs.
-    - name: Create systemd unit to add public route to br-ctlplane
-      template:
-        src: "standalone_public_route.service.j2"
-        dest: "/etc/systemd/system/standalone_public_route.service"
-        owner: root
-        group: root
-        mode: '0644'
-      vars:
-        interface_name: br-ctlplane
-        cidr: "{{ public_cidr }}"
-
-    - name: Create systemd unit to add SNAT rule for public network
-      template:
-        src: "standalone_public_snat.service.j2"
-        dest: "/etc/systemd/system/standalone_public_snat.service"
-        owner: root
-        group: root
-        mode: '0644'
-      vars:
-        cidr: "{{ public_cidr }}"
-        exit_if: "{{ ansible_default_ipv4.interface }}"
-        exit_ip: "{{ ansible_default_ipv4.address }}"
+  - name: Create systemd unit to add SNAT rule for hostonly network
+    template:
+      src: "standalone_snat.service.j2"
+      dest: "/etc/systemd/system/standalone_hostonly_snat.service"
+      owner: root
+      group: root
+      mode: '0644'
+    vars:
+      cidr: "{{ hostonly_cidr }}"
+      exit_if: "{{ network_info.public_ipv4.interface }}"
+      exit_ip: "{{ network_info.public_ipv4.address }}"
 
-    - name: Activate systemd units
-      systemd:
-        name: "{{ item }}.service"
-        enabled: yes
-        state: started
-        daemon_reload: yes
-      loop:
-      - standalone_public_route
-      - standalone_public_snat
-    when: not public_uses_external_nic
+  - name: Activate systemd units
+    systemd:
+      name: "{{ item }}.service"
+      enabled: yes
```

Comment:
yes is deprecated everywhere in ansible, FYI, we use booleans now, here "true"

Reply:
TIL!

That will take as long as python2 deprecation to clean up at this point, though 😂